.doks/values.yaml

```diff
@@ -91,3 +91,6 @@ global:
     hostname: "bitbucket-server.codacy.org"
     protocol: "http"
     port: 7990
+    consumerKey: "codacy-stash-link"
+    consumerPublicKey: "CHANGE_ME"
+    consumerPrivateKey: "CHANGE_ME"
```

Comment:
You could expose this through the .doks/Makefile on the helm command:
--set global.bitbucketenterprise.consumerPublicKey=$${BITBUCKET_ENTERPRISE_PUBLIC_KEY}
--set global.bitbucketenterprise.consumerPrivateKey=$${BITBUCKET_ENTERPRISE_PRIVATE_KEY}